.circleci/config.yml

```diff
@@ -7,54 +7,42 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
-      - image: circleci/python:2.7
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
+      - image: wackerbarth/builder:18.04
```

Comment:
There's not a lot I can meaningfully review here - what does CircleCI currently do for us? Is it just styling (and if so, what languages), or do you have unittests running?